src/python/grpcio_tests/tests/interop/client.py

```diff
@@ -51,6 +51,10 @@ def parse_interop_client_args():
                         default=False,
                         type=resources.parse_bool,
                         help='replace platform root CAs with ca.pem')
+    parser.add_argument('--custom_credentials_type',
+                        choices=["compute_engine_channel_creds"],
+                        default=None,
+                        help='use google default credentials')
```

Comment:
Is there going to be another PR updating our interop script?

Reply:
The interop tests for this actually live in google3 because there are only certain GCP projects where DirectPath is enabled. I'll be following up with a CL for that.